movieDetail.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Movie Box</title>
+    <link rel="stylesheet" href="./assests/styleSheets/movieDetail.css" />
+  </head>
+  <body>
+    <main>
+      <div class="details-container">
+        <div class="cover-image-and-webtitle">
+          <h1>Movies</h1>
+          <img
+            src="./assests/images/luca-cover.jpeg"
+            alt="luca animation cover image"
+            class="cover-image"
+          />
+        </div>
+        <div class="detail-text">
```

Comment:
why is this div needed? can't we remove it?

Reply:
it's for styling all text part.
in mobile changed it to `section` cause without this ,it's children are setting as row